js/menus/menuTemplate.js

```diff
@@ -232,12 +233,29 @@ function getTemplate(app) {
         }
     )
 
-    // Window menu -> minimizeOnClose.
+    // Window menu -> alwaysOnTop.
+    template[index].submenu.push(
+        {
+            label: 'Always on top',
+            type: 'checkbox',
+            checked: alwaysOnTop,
+            click: (item) => {
+                alwaysOnTop = item.checked;
+                let browserWin = electron.BrowserWindow.getAllWindows();
```

Comment:
i see this as functionality that should be provided by windowMgr code.  we should see about moving that there, but i see this would cause circular dep, so need to think about how to do this?  maybe some refactoring needed.